version/version.go

```diff
@@ -1,5 +1,30 @@
 package version
 
+import (
+	"fmt"
+	"runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+)
+
+var log = logf.Log.WithName("version")
+
+//var needs to be used instead of const for ldflags
```

Comment:
I don't think you need a comment here - This cannot be const as we will replace values....